cocos/2d/CCFontAtlas.h

```diff
@@ -139,6 +141,7 @@ class CC_DLL FontAtlas : public Ref
     void* _iconv;
 
     // Dynamic GlyphCollection related stuff
+    Texture2D::PixelFormat _currentPixelFormat;
```

Comment:
It seems `_currentPixelFormat ` is not needed.

Reply:
Yes, I will remove it